public/javascripts/TimestampLocalization.js

```diff
@@ -0,0 +1,26 @@
+//changes timestamps from UTC to local time. Updates any data order variables for tables.
```

Comment:
Can you reformat your comments to be of the form
// This is a comment.
So it starts with a space and a capital letter then ends with a period.
I really need to share a style guide with everyone :sweat: